### PR TITLE
Update bookkeepers.md with http bookkeeper configuration information.

### PR DESCRIPTION
**Pull request by powerpbx**

### CODE REVIEW

File: core/kazoo_services/doc/bookkeepers.md [hunk 1]
```diff
@@ -24,7 +24,57 @@ The first [bookkeeper module](https://github.com/2600hz/kazoo/blob/master/core/k
...
+       "sync_buffer_period": 600
+   },
+   ```
+After refreshing the db settings and adding a SIP device, Kazoo will try contact the remote bookkeeper with the updates after approximately 600 seconds. 
+   
+Here is some sample PHP code for testing the collection of data on the bookkeeper end.
+```php
+<?php
+
+require ('init.php');
```
Comment (fenollp) [inline core/kazoo_services/doc/bookkeepers.md]: Leftover?

Comment (powerpbx) [inline core/kazoo_services/doc/bookkeepers.md]: I do not understand.

File: core/kazoo_services/doc/bookkeepers.md [hunk 2]
```diff
@@ -24,7 +24,57 @@ The first [bookkeeper module](https://github.com/2600hz/kazoo/blob/master/core/k
...
+"default": {
+       "sync_services": true,
+       "support_billing_id": true,
+       "should_save_master_audit_logs": false,
+       "master_account_bookkeeper": "kz_bookkeeper_http",
+       "scan_rate": 20000,
+       "sync_buffer_period": 600
+   },
+   ```
+After refreshing the db settings and adding a SIP device, Kazoo will try contact the remote bookkeeper with the updates after approximately 600 seconds.
```
Comment (lazedo) [inline core/kazoo_services/doc/bookkeepers.md]: this is not accurate. the 600 seconds is an idle period, ie, if you make a change and another after 400 seconds, the 600 seconds count will be reset at 2nd change making the sync only to be excuted after ~1000 seconds from 1st change if there are no other changes after the 2nd.

Comment (powerpbx) [inline core/kazoo_services/doc/bookkeepers.md]: Perhaps you should document that.

File: core/kazoo_services/doc/bookkeepers.md [hunk 3]
```diff
@@ -24,7 +24,57 @@ The first [bookkeeper module](https://github.com/2600hz/kazoo/blob/master/core/k
...
+"default": {
+       "sync_services": true,
+       "support_billing_id": true,
+       "should_save_master_audit_logs": false,
+       "master_account_bookkeeper": "kz_bookkeeper_http",
+       "scan_rate": 20000,
+       "sync_buffer_period": 600
+   },
+   ```
+After refreshing the db settings and adding a SIP device, Kazoo will eventually try contact the remote bookkeeper with the new information.
```
Comment (lazedo) [inline core/kazoo_services/doc/bookkeepers.md]: change `Kazoo will eventually try contact` with `Kazoo will contact`

File: core/kazoo_services/doc/bookkeepers.md [hunk 4]
```diff
@@ -24,7 +24,57 @@ The first [bookkeeper module](https://github.com/2600hz/kazoo/blob/master/core/k
...
+"default": {
+       "sync_services": true,
+       "support_billing_id": true,
+       "should_save_master_audit_logs": false,
+       "master_account_bookkeeper": "kz_bookkeeper_http",
+       "scan_rate": 20000,
+       "sync_buffer_period": 600
+   },
+   ```
+After refreshing the db settings and adding a SIP device, Kazoo will contact the remote bookkeeper with the new information.
```
Comment (lazedo) [inline core/kazoo_services/doc/bookkeepers.md]: @powerpbx thanks for the update. after reviewing it again, i think the `After refreshing the db settings` is not accurate. it should be `After assigning the http bookkeeper to the master account, whenever a billable change occurs (device, user, ...), Kazoo will .... ` 

there should be no need to refresh db settings as they should be automatically be used after the creation/update of the configuration/services docs.

**Comment by jamesaimonetti** [comment]
@powerpbx merged into master but github doesn't seem to track rebased commits (You can see the commit [here](https://github.com/2600hz/kazoo/commit/2dc591481ae53cd6a66628f241fa927d2cf74e36) )

**Pull request closed without merging**